src/Workspaces/MSBuild/Core/MSBuild/MSBuildProjectLoader.cs

```diff
@@ -180,7 +181,11 @@ public async Task<SolutionInfo> LoadSolutionInfoAsync(
             SetSolutionProperties(absoluteSolutionPath);
         }
 
-        var buildHostProcessManager = new BuildHostProcessManager(Properties, loggerFactory: _loggerFactory);
+        var binLogPathProvider = IsBinaryLogger(msbuildLogger)
```

Comment:
In method signature, there was `#pragma warning disable IDE0060` that should no longer be necessary.